include/userobjects/ChargeAccumulator.h

```diff
@@ -0,0 +1,46 @@
+//* This file is part of FENIX: Fusion Energy Integrated Multiphys-X,
+//* A multiphysics application for modeling plasma facing components
+//* https://github.com/idaholab/fenix
+//*
+//* FENIX is powered by the MOOSE Framework
+//* https://www.mooseframework.inl.gov
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+#include "GeneralUserObject.h"
+#include "ResidualAccumulator.h"
+#include "PICStudyBase.h"
```

Comment:
```suggestion

class PICStudyBase;
```

Should be sufficient + including in the source

Reply:
I have made this change but I also had to include "Ray.h" after removing "#include PICStudyBase.h" since this class has a `RayDataIndex` member variable.